src/NServiceBus.Persistence.AzureStorage/Timeout/TimeoutLogic/TimeoutPersister.cs

```diff
@@ -151,79 +138,28 @@ public TimeoutPersister(string timeoutConnectionString, string timeoutDataTableN
             var now = DateTime.UtcNow;
 
             var timeoutDataTable = client.GetTableReference(timeoutDataTableName);
-            var timeoutManagerDataTable = client.GetTableReference(timeoutManagerDataTableName);
 
-            await TryUpdateSuccessfulRead(timeoutManagerDataTable).ConfigureAwait(false);
-
-            var lastSuccessfulReadEntity = await GetLastSuccessfulRead(timeoutManagerDataTable).ConfigureAwait(false);
-            var lastSuccessfulRead = lastSuccessfulReadEntity?.LastSuccessfullRead;
-
-            TableQuery<TimeoutDataEntity> query;
-
-            if (lastSuccessfulRead.HasValue)
-            {
-                query = new TableQuery<TimeoutDataEntity>()
-                    .Where(
+            var query = new TableQuery<TimeoutDataEntity>()
+                .Where(
+                    TableQuery.CombineFilters(
                         TableQuery.CombineFilters(
-                            TableQuery.CombineFilters(TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.GreaterThanOrEqual, lastSuccessfulRead.Value.ToString(partitionKeyScope)),
-                                TableOperators.And,
-                                TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.LessThanOrEqual, now.ToString(partitionKeyScope))),
+                            TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.GreaterThanOrEqual, now.AddYears(-1).ToString(partitionKeyScope)),
                             TableOperators.And,
-                            TableQuery.GenerateFilterCondition("OwningTimeoutManager", QueryComparisons.Equal, endpointName))
-                    );
-            }
-            else
-            {
-                query = new TableQuery<TimeoutDataEntity>()
-                    .Where(TableQuery.GenerateFilterCondition("OwningTimeoutManager", QueryComparisons.Equal, endpointName));
-            }
+                            TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.LessThanOrEqual, now.ToString(partitionKeyScope))),
+                        TableOperators.And,
+                        TableQuery.GenerateFilterCondition("OwningTimeoutManager", QueryComparisons.Equal, endpointName))
+                );
 
             var timeoutDataEntities = await timeoutDataTable.ExecuteQueryAsync(query, take: 1000).ConfigureAwait(false);
-            var result = timeoutDataEntities.OrderBy(c => c.Time);
-
-            var allTimeouts = result.ToList();
-            if (allTimeouts.Count == 0)
-            {
-                return new TimeoutsChunk(new TimeoutsChunk.Timeout[0], now.AddSeconds(1));
-            }
 
-            var pastTimeouts = allTimeouts.Where(c => c.Time > startSlice && c.Time <= now).ToList();
-            var futureTimeouts = allTimeouts.Where(c => c.Time > now).ToList();
+            var timeouts = timeoutDataEntities.Where(c => c.Time <= now).OrderBy(c => c.Time).ToList();
```

Comment:
I treat this as `startSliceProposedByTheCore`. AFAIK Raven persistence scans the past from time to time as well, doesn't it?